index.html

```diff
@@ -15,19 +15,23 @@ <h2>Overview</h2>
          <table>
     <tr>
         <td><img src="https://mk0itemsssrcorgqb419.kinstacdn.com/wp-content/uploads/2019/03/iStock-165790610-1000x569.jpg" width="350" height="300"></td>
-        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideology based website that focuses on describing the pros/cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. Philosophy will further look at the influence of the age of enlightment, the Roman influence of early Afro-European politics and nationalites.</p></td>
+        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideology based website that focuses on describing the pros and cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. The philosophy of political figures will further look at the influence of the age of enlightment and the Roman influence of early Afro-European politics and nationalites.</p></td>
```

Comment:
Who are the political figures in this? ` The philosophy of political figures ` ?